### PR TITLE
feat: add marks to slider

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Add marks to Slider**
**Jira Ticket:** https://makingsense.atlassian.net/browse/DAT-1155

**Before:** 
Captura de Pantalla 2022-10-11 a la(s) 1 43 57 p  m
Captura de Pantalla 2022-10-11 a la(s) 1 44 11 p  m

**After:**
Captura de Pantalla 2022-10-11 a la(s) 1 44 39 p  m
Captura de Pantalla 2022-10-11 a la(s) 1 45 02 p  m

### CODE REVIEW

File: src/documentation/templates/plan-calculator.html [hunk 1]
```diff
@@ -232,14 +242,29 @@ <h4>Contactos</h4>
...
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
```
Comment (andresmoschini) [inline src/documentation/templates/plan-calculator.html:263]: Why did you keep these empty?

**Comment by andresmoschini** [comment]
👍

File: src/documentation/templates/plan-calculator.html (hunk 1 above)
Comment (jcamposmk) [reply src/documentation/templates/plan-calculator.html:263]: in case there are more than 8 plans to be shown in the slider

Comment (jcamposmk) [reply src/documentation/templates/plan-calculator.html:263]: Captura de Pantalla 2022-10-11 a la(s) 1 50 03 p  m

Comment (andresmoschini) [reply src/documentation/templates/plan-calculator.html:263]: Thanks!

A related question, should not the number of plans be dynamic or data based?

Comment (jcamposmk) [reply src/documentation/templates/plan-calculator.html:263]: Yes, it's dynamic

Comment (andresmoschini) [reply src/documentation/templates/plan-calculator.html:263]: If it is dynamic, why did you add 13 `li` manually?

Comment (jcamposmk) [reply src/documentation/templates/plan-calculator.html:263]: Sorry, they are harcoded in style guide ... 
But, they will be dynamic in the webapp.

**Comment by GusBaamonde** [comment]
:rocket:

**Pull request merged**